apps/server/src/modules/authorization/authorization.service.ts

```diff
@@ -61,6 +63,27 @@ export class AuthorizationService extends BasePermissionManager {
 		}
 	}
 
+	async hasPermissionsByReferences(
+		userId: EntityId,
+		entityName: AllowedAuthorizationEntityType,
+		entityId: EntityId,
+		permissions: Permission[]
+	): Promise<Map<Permission, boolean>> {
+		const returnMap: Map<Permission, boolean> = new Map();
+		await Promise.all(
+			permissions.map(async (perm) => {
+				const ret = await this.hasPermissionByReferences(
+					userId,
+					entityName,
+					entityId,
+					PermissionContextBuilder.permissionOnly([perm])
+				);
+				returnMap.set(perm, ret);
+			})
+		);
+		return returnMap;
+	}
+
 	async checkPermissionByReferences(
```

Comment:
Same here. No need for authorization by reference.

Reply:
See other comment on this file